src/platforms/javascript/guides/ember/ember2.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Ember 2.x
+redirect_from:
+  - /clients/javascript/integrations/ember/
+---
+
+To use Sentry with your Ember application, you will need to use Sentry’s browser JavaScript SDK: `@sentry/browser`.
+
+On its own, `@sentry/browser` will report any uncaught exceptions triggered from your application.
+To use ESM imports without any additional configuration, you can use `ember-auto-import`
+by installing it with `ember install ember-auto-import`.
+
+Starting with version `5.x` our `Ember` integration lives in its own `@sentry/integrations` package.
+Install the package using either `npm` or `yarn`:
+
+```bash
+# Using yarn
+yarn add @sentry/integrations
+
+# Using npm
+npm install @sentry/integrations
+```
+
+Then add to your `app.js`:
+
+```javascript
+import * as Sentry from "@sentry/browser";
+import { Ember as EmberIntegration } from "@sentry/integrations";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [new EmberIntegration()],
+});
+```
+
+If you are using the CDN version or the Loader, we provide a standalone file for every integration, which you can use as follows:
```

Comment:
We moved the CDN content out of the core JS Getting Started, so feel free to point to that as appropriate - https://docs.sentry.io/platforms/javascript/configuration/install-cdn/